tests/TDD/Chapter1/Chapter1Test.php

```diff
@@ -2,9 +2,13 @@
 
 namespace TDD\Chapter1;
 
-class Chapter1Test extends \PHPUnit_Framework_TestCase
+class Chapter1Test extends TestCase 
 {
-    public function testMultiplication()
+
+	/**
+	 * Tests that Dollar's public variable, $amount, is the same as the given amount
+	 */
```

Comment:
There's some extra space between the start of the function declaration and the comments.